source/admin_manual/list_of_events.rst

```diff
@@ -955,7 +955,7 @@ requests (e.g. doveadm HTTP API).
 POP3
 ====
 
-.. dovecot_event:field_group:: pop3_client
+.. dovecot_core:event:: pop3_client
```

Comment:
I think this was correct originally. pop3_client isn't an event, it's a group of settings.

Reply:
But the section was/is empty currently. The heading is displayed but the field_group was not displayed. So even if this is an incorrect patch, something should be done about it I'd say.

Reply:
It's not just pop3. All the field_groups seem to have broken. I guess with Michael's patch that did the big change..

Reply:
Michael clarified in another pull request that it's supposed to be hidden. Only the fields listed in the group are visible in the events that inherit from it.

Reply:
I changed it now to hide the POP3 section, but not change the field_group itself. I think there is additional issues with this page as was discussed in the meeting this morning (regarding the events on Wed 15. March 2023).